index.html

```diff
@@ -83,7 +83,7 @@ <h1>Page content</h1>
         </p>
       </div>
     </main>
-    <script src="dist/index.js"></script>
+    <script src="dist/global-nav.js"></script>
```

Comment:
Has the interface changed? Or was this just wrong before?

Reply:
The interface was updated and published as a version release but no one updated the interface on the local demo.